src/commands/acl-setuser.json

```diff
@@ -0,0 +1,43 @@
+{
+    "SETUSER": {
+        "summary": "Modify or create the rules for a specific ACL user",
+        "complexity": "O(N). Where N is the number of rules provided.",
+        "group": "server",
+        "since": "6.0.0",
+        "arity": -3,
+        "container": "ACL",
+        "function": "aclCommand",
+        "history": [
+            [
+                "6.2",
+                "Added Pub/Sub channel patterns."
+            ]
+        ],
+        "command_flags": [
+            "admin",
+            "noscript",
+            "loading",
+            "stale",
```

Comment:
`stale` and `loading` are misleading, these used to be `ok-stale` or `allow-stale`

Reply:
we discussed this and reached the conclusion that it's better if the input flags (in the json files) are the same as the output flags (in COMMAND)

do you wish to re-open that discussion?

Reply:
ohh, i remember.
so the flags in COMMAND are already part of redis API, and we want to align with them..